### PR TITLE
added actual workshops

### LINKED ISSUE #134: Add actual workshops to workshop page

### PR DESCRIPTION
**Pull request by eric8yang**
Fixes: #134

button is not actually on the top bar yet, can add later when needed

### CODE REVIEW

**Comment by jodymlin** [comment]
Is the Workshops button supposed to be added to the navbar in this PR? I didn't see it in the preview

File: src/components/WorkshopPage/WorkshopPage.js [hunk 1]
```diff
@@ -11,39 +11,139 @@ const workshops = [
 		type: 'Web Development',
 		elements: [
 			{
-				title: 'Web Dev #1: HTML CSS JS',
+				title: 'Intro to HTML',
 				author: 'Jakob Reinwald',
-				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
-				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+				description: `HTML(Hyper Text Markup Language) is a fancy way of saying text
```
Comment (TimothyGu) [inline src/components/WorkshopPage/WorkshopPage.js:16]:
```suggestion
				description: `HTML (HyperText Markup Language) is a fancy way of saying text
```

**Review by TimothyGu — COMMENTED** [review]
Looks good, though let's add the workshop button too.

**Review by TimothyGu — APPROVED** [review]

**Comment by TimothyGu** [comment]
Marking as Draft to make it clear we are merging it right when HOTH starts

**Pull request merged**